spec/apollo-federation/service_field_v2_spec.rb

```diff
@@ -1539,6 +1539,50 @@ def self.visible?(context)
       )
     end
 
+    it 'returns SDL that inherits object directives' do
+      base_object_with_id = Class.new(base_object) do
+        key fields: 'id'
+
+        field :id, String, null: false
+      end
+
+      product = Class.new(base_object_with_id) do
+        graphql_name 'Product'
+      end
+
+      schema = Class.new(base_schema) do
+        federation version: '2.0'
+        orphan_types product
+      end
+
+      expect(execute_sdl(schema)).to match_sdl(
+        <<~GRAPHQL,
+          extend schema
+            @link(url: "https://specs.apollo.dev/federation/v2.3", import: ["@inaccessible", "@tag"])
+
+          type Product @federation__key(fields: "id") {
+            id: String!
+          }
+        GRAPHQL
+      )
+    end
+
+    it 'returns SDL that inherits schema directives' do
+      new_base_schema = Class.new(base_schema) do
+        federation version: '2.0', link: { as: 'fed2' }
```

Comment:
Since you are adding a custom link namespace, can you add an example to the test so we can see that directives are being prefixed with `fed2__` instead of the default?